app/modules/content-items/components/EditorBlock.js

```diff
@@ -26,7 +26,11 @@ type StateProps = {
 type Props = TranslatorProps & PassedProps & StateProps;
 
 const mapStateToProps = (state: State, props: PassedProps): StateProps => {
-  const contentItem = getById(state, props.contentItemId);
+  const contentItem = getById(state, { id: props.contentItemId });
+
+  if (contentItem == null) {
+    throw new Error(`ContentItem with id "${props.contentItemId}" could not be found.`);
```

Comment:
Can we figure out an `Error` hierarchy here? That would allow us to catch only the right errors, and add precise i18n as well.

Reply:
^ That seems a bit to big to fix in this pull request. Can you make an issue instead?

Reply:
Yep, that's why I approved this PR. See #27.